src/components/AlertLine.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <li 
+    class="alert" 
+    :style="{ 'background-color': alert.backgroundColor }">
+    <div class="icon-container">
+      <img class="icon" :src="`img/${topic}.svg`" />
+    </div>
+    <div class="alert-message">{{ alert.message }}</div>
+    <div class="alert-date">{{ alert.time }}</div>
+    <a @click="toggleCollapse" href="#" class="icon-container">
+      <img v-if="collapsed" class="icon" src="img/down-arrow.svg" />
+      <img v-else class="icon" src="img/up-arrow.svg" />
+    </a>
+    <div v-if="!collapsed" class="alert-details">
+      <h3>Server Information</h3>
+      <ul class="server-info">
+        <li>FQDN: {{alert.fqdn}}</li>
+        <li>Hostname: {{alert.serverInfo.Hostname}}</li>
+        <li>ClusterID: {{alert.serverInfo.ClusterID}}</li>
+        <li>ServerID: {{alert.serverInfo.ClusterID}}</li>
+      </ul>
+    </div>
+    <div v-if="!collapsed" class="alert-buttons">
+      <button @click="snoozeAlert">Snooze Alert</button>
+      <button @click="openGraphana">Open in Grafana</button>
+    </div>
+  </li>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { Component, Prop } from 'vue-property-decorator';
+
+import Alert from '@/interfaces/Alert';
+
+@Component
+export default class AlertLine extends Vue {
+  // region properties
+  @Prop(Object) public alert!: Alert;
+  @Prop(String) public topic!: string;
+  // endregion
+
+  // region public members
+  // endregion
+
+  // region public methods
+  // endregion
+
+  // region private members
+  private collapsed: boolean = true;
+  // endregion
+
+  // region constructor
+  // endregion
+
+  // region private methods
+  private toggleCollapse(): void {
+    this.collapsed = !this.collapsed;
+  }
+
+  private snoozeAlert(): void {
+    // Snooze alert using backend
+  }
+
+  private openGraphana(): void {
+    window.open('https://dev.xikolo.de/grafana/d-solo/000000001/generic-physical-host?orgId=1&var-fqdn=ctrl-dev.os.openhpi.de&panelId=1', '_blank')
```

Comment:
Is the fqdn hard coded by purpose?
And also, it always shows the `cpu` panel 🤔

Reply:
Changed that to the fqdn relating to the current alert.
I also removed the panel id to show the whole dashboard. Anyways, we should think about adding dashboard and panel id to the kapacitor event information.

Reply:
Yes, you're right. We will do so in the next Kapacitor "release" 📦  :p